docs/source/getting_started/installation.rst

```diff
@@ -64,41 +64,42 @@ You can also build and install vLLM from source:
 
 .. code-block:: console
 
-    $ git clone https://github.com/vllm-project/vllm.git
-    $ cd vllm
-    $ pip install -e .  # This may take 5-10 minutes.
+   $ git clone https://github.com/vllm-project/vllm.git
+   $ cd vllm
+   $ pip install -e .  # This may take 5-10 minutes.
 
 .. tip::
 
-    Building from source requires quite a lot compilation. If you are building from source for multiple times, it is beneficial to cache the compilation results. For example, you can install `ccache <https://github.com/ccache/ccache>`_ via either `conda install ccache` or `apt install ccache` . As long as `which ccache` command can find the `ccache` binary, it will be used automatically by the build system. After the first build, the subsequent builds will be much faster.
+   Building from source requires quite a lot of compilation. If you are building from source for multiple times, it is beneficial to cache the compilation results. For example, you can install `ccache <https://github.com/ccache/ccache>`_ via either `conda install ccache` or `apt install ccache`. As long as the `which ccache` command can find the `ccache` binary, it will be used automatically by the build system. After the first build, the subsequent ones will be much faster.
 
 .. tip::
-    To avoid your system being overloaded, you can limit the number of compilation jobs
-    to be run simultaneously, via the environment variable `MAX_JOBS`. For example:
 
-    .. code-block:: console
+   To avoid your system being overloaded, you can limit the number of compilation jobs to be run simultaneously via the `MAX_JOBS` environment variable. For example:
```

Comment:
```suggestion
   To avoid your system being overloaded, you can limit the number of compilation jobs to be run simultaneously via the ``MAX_JOBS`` environment variable. For example:
```